layouts/partials/nav.html

```diff
@@ -35,13 +35,15 @@
       </li>
 
       {{ if $isHere }}
+      {{ if ne .TableOfContents "<nav id=\"TableOfContents\"></nav>" }} <!-- not render ToC when is empty -->
```

Comment:
I'm unsure why this extra `if` is necessary. I think that the `if` on the line below (line 39) serves the same purpose -- though the magic number "32" might be off and need to be adjusted.

Reply:
Actually, 32 seems to work just fine according to my tests.

Reply:
Yes, it's working using 32. 

I was thinking about asking to replace the new `if` for the other one. IMHO, the comparison with the empty nav code is more clear than counting characters. But I have removed it for now.